### PR TITLE
[#43123] Remove files storages feature flag

### PR DESCRIPTION
**Pull request by apfohl**
https://community.openproject.org/work_packages/43123

### CODE REVIEW

File: modules/storages/spec/features/storages_module_disabled_spec.rb [hunk 1]
```diff
@@ -59,13 +59,6 @@
     Capybara.raise_server_errors = old_value
   end
 
-  it 'removes storages from administration and project pages' do
```
Comment (wielinde) [inline modules/storages/spec/features/storages_module_disabled_spec.rb:62]: @apfohl Removing these lines makes the whole file not do anything anymore, I believe. So we either remove it completely or we find stuff in here, we want to have covered even without the feature flag present. For example convering the consequences of activating and disabling the File storages module per project.

**Review by wielinde — CHANGES_REQUESTED** [review]
@apfohl I skimmed over this PR. It looks great. I found just one issue with the spec covering enabling and disabling the module. I didn't look into the details, though. Once that is resolved, feel free to merge.

File: modules/storages/spec/features/storages_module_disabled_spec.rb (hunk 1 above)
Comment (apfohl) [reply modules/storages/spec/features/storages_module_disabled_spec.rb:62]: I left the part in there that test activated case.

Comment (apfohl) [reply modules/storages/spec/features/storages_module_disabled_spec.rb:62]: I refactored the spec and added a disabled module spec.

File: modules/storages/spec/features/storages_module_spec.rb [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+require_relative '../spec_helper'
+
+describe 'Storages module', type: :feature, js: true do
+  current_user { create(:admin) }
+
+  let(:role) { create(:role, permissions: %i[manage_storages_in_project select_project_modules edit_project]) }
+  let(:storage) { create(:storage, name: "Storage 1") }
+  let(:project) { create(:project, enabled_module_names: %i[storages work_package_tracking]) }
+
+  # around do |example|
```
Comment (wielinde) [inline modules/storages/spec/features/storages_module_spec.rb]: @apfohl Does this commented block serve a purpose that I don't understand or can we remove it?

Comment (apfohl) [reply modules/storages/spec/features/storages_module_spec.rb]: It was a left over. I removed it.

**Review by wielinde — APPROVED** [review]
LGTM! Tx.

**Pull request merged**